src/components/pages/dashboard/index.tsx

```diff
@@ -460,6 +462,8 @@ const Dashboard = (): any => {
     'availableTo_asc',
   ];
 
+  const shouldShowExtraInfo = useMatchBreakpoint(Breakpoints.L);
```

Comment:
No way to achieve this with CSS? This will only be triggered on the client-side now, causing a difference between the server and client render.

Reply:
I think we could achieve similar results using something like 
`display={{
  default: 'flex',
  l: 'hidden'
}}`